torch/fx/graph.py

```diff
@@ -357,6 +350,19 @@ def type_repr(o : Any):
             # Common case: this is a regular module name like 'foo.bar.baz'
             return add_global(typename, o)
 
+        def _format_args(args: Tuple[Argument, ...], kwargs: Dict[str, Argument]) -> str:
+            def _get_repr(arg):
+                if isinstance(arg, tuple) and type(arg) != tuple:
```

Comment:
I think it's possible that there are tuple subclasses that are not `NamedTuple`. Can we make this more specific by checking if there's a `_fields` attribute: https://stackoverflow.com/a/2166841 (this solution for checking for a `NamedTuple` is ridiculous but it's python's fault lol)

Reply:
Done.